sky/templates/kubernetes-ray.yml.j2

```diff
@@ -279,6 +279,7 @@ available_node_types:
             # Identifies the SSH jump pod used by this pod. Used in life cycle management of the ssh jump pod.
             skypilot-ssh-jump: {{k8s_ssh_jump_name}}
             skypilot-user: {{ user }}
+            skypilot-original-cluster-name: {{cluster_name}}
```

Comment:
This won't work with `cluster_name` that does not follow the regex required by kubernetes will fail. The reason we used `cluster_name_on_cloud` is to prevent the case. Why do we need to set the `cluster_name` in labels.

Reply:
This is to show the clusters using the volumes, and update the solution to get from DB.